FirebaseFunctions/Sources/Public/FirebaseFunctions/FIRHTTPSCallable.h

```diff
@@ -47,12 +47,12 @@ NS_SWIFT_NAME(HTTPSCallable)
  * Executes this Callable HTTPS trigger asynchronously without any parameters.
  *
  * The request to the Cloud Functions backend made by this method automatically includes a
- * Firebase Instance ID token to identify the app instance. If a user is logged in with Firebase
+ * Firebase Installations token to identify the app instance. If a user is logged in with Firebase
  * Auth, an auth ID token for the user is also automatically included.
  *
- * Firebase Instance ID sends data to the Firebase backend periodically to collect information
- * regarding the app instance. To stop this, see `[FIRInstanceID deleteIDWithHandler:]`. It
- * resumes with a new Instance ID the next time you call this method.
+ * Firebase Installations sends data to the Firebase backend periodically to collect information
+ * regarding the app instance. To stop this, see `Installations.delete()`. It
```

Comment:
Hm, actually Functions use FCM token now. FCM token will also be refreshed when `Installations.delete()` is called but maybe we should be more precise here. @chliangGoogle WDYT?

Reply:
Yeah and if function uses token, then they also need to call Messaging.deleteData() to remove FCM token info.
https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessaging.h#L397